pkg/collector/corechecks/system/disk/iostats.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 const (
-	// SectorSize is exported in github.com/shirou/gopsutil/disk (but not working!)
+	// SectorSize is exported in github.com/shirou/gopsutil/v3/disk (but not working!)
 	SectorSize       = 512
```

Comment:
Is this still not working?

Reply:
@djmitche Thank you. No idea if it is working. I was replacing all references to implicit import to explicit. I just changed a comment as well. Not sure if it was correct thing to do but I think it was.

Reply:
It looks like it's _not_ exported, so maybe update the comment?

Reply:
@djmitche and @albertvaka what should I do about it. He is right the comment is not correct anymore. The const is not exported in v3. It does not break anything and it is only inaccurate comment with the same consequences. The problem that the PR is is merged already. Should I create backlog jira to change the comment alone?

Reply:
It doesn't look like it's merged..

<img width="109" alt="image" src="https://user-images.githubusercontent.com/28673/159712869-1ce20bcb-c446-4395-9bf5-19d84ff202c8.png">

But if that's incorrect, then just make another PR to change the comment - no need for a backlog card, sprint planning, etc.

Reply:
You are right! Damn. it was a while ago I was waiting for everyone's approval and shifted my focus on other issue. I thought that that I have merged it already. Will do it soon. Thank you again.

Reply:
Please see my last commit which should address your comment. Thank you.

Reply:
Strange, change a comment and now lot of things are failing :(

Reply:
... plus conflicts